tsdb/engine/tsm1/float.go

```diff
@@ -182,20 +178,29 @@ func (it *FloatDecoder) Next() bool {
 	}
 
 	// read compressed value
-	bit, err := it.br.ReadBit()
-	if err != nil {
+	var bit bool
+	if it.br.CanReadBitFast() {
+		bit = it.br.ReadBitFast()
+	} else if v, err := it.br.ReadBit(); err != nil {
 		it.err = err
 		return false
+	} else {
+		bit = v
 	}
```

Comment:
This is the usage for trying the fast bit read first and then falling back to `ReadBit()`.